torch/csrc/jit/script/module.h

```diff
@@ -481,6 +482,23 @@ struct Module {
     }
     fn(*this);
   }
+  void train(bool on = true) {
+    for (auto& submod : get_modules()) {
+      submod->module->train(on);
+    }
+    register_parameter("training", torch::tensor(on ? 1 : 0, at::kLong), /*is_buffer=*/true);
+  }
+  void eval() {
+    train(/*on=*/false);
+  }
+  bool is_training() {
+    if (auto p = find_parameter("training")) {
+      return (*p->slot()).item<int64_t>() == 1;
+    } else {
```

Comment:
nit: remove else block, just `return true` + comment (since you return in the `if` block)